tests/extensions/inline/sub/full/invalid/extra-content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../extra-content.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../extra-content.rng" schematypens="http://purl.oclc.org/dsdl/schematron"?>
```

Comment:
This case/schema doesn't do anything anymore as it no longer uses `libero.inline.full` (`unknown-element` already covers there being an unknown element).

Reply:
As I understand it we can get rid of the following tests:

- tests/extensions/inline/sup/full/invalid/extra-content.xml
- tests/extensions/inline/sup/full/invalid/nested.xml
- tests/extensions/inline/sup/full/invalid/nested-deep.xml
- tests/extensions/inline/sup/limited/invalid/extra-content.xml
- tests/extensions/inline/sup/limited/invalid/nested.xml
- tests/extensions/inline/sup/limited/invalid/nested-deep.xml

All the above cases/schemas are covered by:

- tests/extensions/inline/sup/full/invalid/unknown-element.xml
- tests/extensions/inline/sup/limited/invalid/unknown-element.xml